package.json

```diff
@@ -57,6 +57,26 @@
           "settings.enabled"
         ]
       },
+      {
+        "permissionName": "ui-organizations.categories.settings.view",
```

Comment:
You don't need to introduce new permission sets.

Reply:
updated